contracts/TrivialToken.sol

```diff
@@ -12,6 +12,7 @@ contract TrivialToken is ERC223Token, PullPayment {
     uint256 constant MIN_BID_PERCENTAGE = 5;
     uint256 constant TOTAL_SUPPLY = 1000000;
     uint256 constant TOKENS_PERCENTAGE_FOR_KEY_HOLDER = 5;
+    uint256 constant CLEANUP_DELAY = 15778463; // 6 months
```

Comment:
Use `180 days` here: http://solidity.readthedocs.io/en/develop/units-and-global-variables.html